### PR TITLE
Modify config for tailwind v2

### PR DESCRIPTION
**Pull request by julianrubisch**
I've still got a problem with purging (of course :-( )

### CODE REVIEW

**Comment by julianrubisch** [comment]
Ah, false alarm... with `NODE_ENV=production`, it works 👍

File: templates/postcss.config.js [hunk 1]
```diff
@@ -1,23 +1,7 @@
...
-
 module.exports = {
   plugins: [
-    require("postcss-import"),
-    require("tailwindcss"),
+    require("tailwindcss")("tailwind.config.js"),
     require("autoprefixer"),
-    ...(process.env.NODE_ENV === "production" ? [purgecss] : []),
-  ],
+    require("postcss-import")
```
Comment (ParamagicDev) [inline templates/postcss.config.js:5]: I believe this should be the first plugin.

https://tailwindcss.com/docs/using-with-preprocessors#build-time-imports

File: templates/tailwind.config.js [hunk 2]
```diff
@@ -1,12 +1,27 @@
 module.exports = {
   future: {
+    defaultLineHeights: true,
+    purgeLayersByDefault: true,
     removeDeprecatedGapUtilities: true,
-    purgeLayersByDefault: false
+    standardFontWeights: true
+  },
+  purge: {
```
Comment (ParamagicDev) [inline templates/tailwind.config.js:8]: I remember there being an issue with Tailwind not purging with the built in purge which is why I moved it to the postcss config. Perhaps @andrewmcodes could shed some light.

**Review by KonnorRogers — COMMENTED** [review]
Good stuff Julian! Thank you for this! Just 2 minor things I'm not 100% on

File: templates/tailwind.config.js (hunk 2 above)
Comment (julianrubisch) [reply templates/tailwind.config.js:8]: I can confirm on netlify it works with NODE_ENV=production

Comment (julianrubisch) [reply templates/tailwind.config.js:8]: image

yep, definitely purges 😅

**Comment by julianrubisch** [comment]
anything I need to do here?

**Comment by ParamagicDev** [comment]
@julianrubisch Nope. Just totally forgot about it sorry!

**Pull request merged**